src/embed.ts

```diff
@@ -94,40 +95,92 @@ async function getVimeoThumbnail(url: URL, sentry: SentryReporter) {
     'https://vimeo.com/api/oembed.json?url=' +
       encodeURIComponent(`https://vimeo.com/${videoId}`)
   )
+  const apiResponseText = await apiResponse.text()
+
+  if (apiResponse.status !== 200) {
+    if (apiResponse.status !== 404) {
```

Comment:
why exclude 404?

Reply:
Vimeo returns a 404 response when the video does not exist (= the video url is wrong). This case is not interesting for us.

Reply:
Hm. It's not a code problem. But we might log it anyway and pass it on to content ppl (or fix it ourselves) since it's still a website problem 😺.
that's what I did for failing routes in the frontend (see https://github.com/serlo/frontend/issues/1254) and it was really  worth it.
Your call.